src/Components/Nav/Nav.js

```diff
@@ -11,8 +26,16 @@ function Nav() {
         </Link>
         <TravelAndActivity>
           <MainNav>
-            <Link>여행</Link>
-            <SubNav>
+            <Link onClick={handleNav} name={'trip'}>
```

Comment:
Link tag로 사용할 게 아닌 것 같습니다! 라우팅을 하지 않네요!

Reply:
```suggestion
            <Link onClick={handleNav} name='trip'>
```
string은 중괄호 없이 사용가능합니다!

Reply:
정답 : Link -> button...맞을까요...? 👀

Reply:
💯